Read version + name from pom.xml for build

timarney opened the pull request:
This should read the version # + artifactId from the pom.xml file and use that for the .jar filename.

Note: We'll need to move this over to a CDS repo to if we want to automate the deployment with Google cloud.

timarney committed:
read version + name from pom.xml for build

pcraig3 reviewed (approved):
This is awesome! 

👍 👍 👍

pcraig3 committed:
Remove commented-out line from Dockerfile

Since the filename is now dynamic, we don't need the old filename
floating around in the Dockerfile.
Plus the version and app name have changed, so it's not even
accurate anyway.
